vidarr/wdl.py

```diff
@@ -207,6 +211,23 @@ def read_output(output: WDL.Decl):
         'workflow': doc.source_text,
         'accessoryFiles': {
             imported.uri: imported.doc.source_text for imported in doc.imports}}
+
+    # Define the output within the workflow
+    for output in doc.workflow.outputs:
+        output_metadata = doc.workflow.meta.get("output_meta", {}).get(output.name, {})
+        if isinstance(
+                output_metadata,
+                dict) and "vidarr_label" in output_metadata:
+            vidarr_label = output_metadata.get("vidarr_label", "")
+            output_value = output.expr
+            if isinstance(output_value, WDL.Expr.Ident):
+                output_value = output_value.name
+
+            # Replace the output definition with Pair[File, Map[String, String]] format
+            workflow['workflow'] = workflow['workflow'].replace(
```

Comment:
The other language features of miniwdl _should_ prevent this, but I am wary of running a 'replace' on the entirely of the workflow text, a line other than the one in the output block might get replaced. Since you have access to a more manageable tree structure using `doc`, I'd recommend seeing if you can move this logic up to within `read_output` or `_map_output` and swapping the WDL.Type for these expressions